src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -1,32 +1,37 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+ * Add support for getting updated context after a transition
+
 5.3
 ---
 
  * Deprecate `InvalidTokenConfigurationException`
- * Added `MermaidDumper` to dump Workflow graphs in the Mermaid.js flowchart format
```

Comment:
Please revert all unrelated changes here. We don't need/want to update all the old changelogs. The convention changed only for new entries.